src/main/java/eu/europa/ted/eforms/xpath/XPathInfo.java

```diff
@@ -0,0 +1,51 @@
+package eu.europa.ted.eforms.xpath;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class XPathInfo {
+	private LinkedList<XPathStep> steps = new LinkedList<>();
```

Comment:
Looks like this needs formatting

Reply:
VS Code keeps switching to indenting with tabs in new files, and a few tabs were left.
Fixed in added commit.